lib/routes/bsky/templates/post.art

```diff
@@ -8,7 +8,14 @@
             <img src="{{ i.fullsize }}" alt="{{ i.alt }}"><br>
         {{ /each }}
     {{ else if embed.$type === 'app.bsky.embed.video#view' }}
-        <img src="{{ embed.thumbnail }}"><br>
+        <video 
+            controls
+            poster="{{ embed.thumbnail }}"
+            style="max-width: 100%; height: auto;"
+            preload="none">
+            <source src="{{ embed.playlist }}" type="application/x-mpegURL">
```

Comment:
> Majority of the browsers (Chrome, Edge & Firefox) don't have native HLS support.

However, this isn't a valid reason to only provide video thumbnails instead of a video player.

How to handle HLS playback should be a consideration for the RSS service provider.